packages/components/src/DateTimePickers/views/HeaderTitle/HeaderTitle.scss

```diff
@@ -6,6 +6,14 @@
 	align-items: center;
 	text-transform: uppercase;
 	font-weight: 600;
+
+	:global(.dropdown-menu) {
+		min-width: 63px;
```

Comment:
why 63 px ?

Reply:
there is no width defined for this dropdown list in guidline. and UX said should be the same to the ActionDropDown. And for now ActionDropDown is a little more than 62px. so I set it 63 here.